daal4py/sklearn/monkeypatch/tests/utils/launch_algorithms.py

```diff
@@ -0,0 +1,108 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+import numpy as np
+import logging
+import random
+
+from daal4py.sklearn import patch_sklearn
+patch_sklearn()
+
+from sklearn.metrics import pairwise_distances, roc_auc_score
+from sklearn.datasets import (
+    make_regression,
+    load_iris,
+    load_diabetes)
+
+import sys
+import pathlib
+absolute_path = str(pathlib.Path(__file__).parent.absolute())
+sys.path.append(absolute_path + '/../')
+from models_info import MODELS_INFO, TYPES
```

Comment:
Add __init__.py file

```suggestion
from .models_info import MODELS_INFO, TYPES
```